cryptofeed/backends/backend.py

```diff
@@ -13,14 +13,14 @@
 
 class BackendBookCallback:
     async def __call__(self, *, feed, pair, book, timestamp):
-        data = {'timestamp': timestamp, 'delta': False, BID: {}, ASK: {}}
+        data = {'feed': feed, 'pair': pair, 'timestamp': timestamp, 'delta': False, BID: {}, ASK: {}}
```

Comment:
the data is already in the write below. If you need the data for rabbitmq, can't you just tweak its writes ?

Reply:
So, why the book data fields didn't include feed and pair, but trades, tick, and funding have? I think this breaks consistency. But I will try to tweak it.